File: gin/entity/po/focus.go
package po

import "time"

type Focus struct {
	ID        uint      `gorm:"column:id" json:"id"`
	UserID    uint      `gorm:"column:user_id" json:"userId"`
	FocusID   uint      `gorm:"column:focus_id" json:"focusId"`
	CreatedAt time.Time `gorm:"column:created_time" json:"createdAt"`
}

func (Focus) TableName() string {
	return "focus"
}

File: fastapi/app/internal/models/focus.py
from datetime import datetime

from app.core.db.base import Base
from sqlalchemy import Column, DateTime, Integer


class Focus(Base):
    """关注实体类"""

    __tablename__ = "focus"
    id = Column(Integer, primary_key=True, index=True, autoincrement=True)
    user_id = Column(Integer, nullable=False)
    focus_id = Column(Integer, nullable=False)
    created_time = Column(DateTime, nullable=True, default=datetime.now)
